docker/nginx/nginx.conf

```diff
@@ -0,0 +1,21 @@
+upstream docker-wiremock {
+    server wiremock:8080;
+}
+
+
+server {
+    listen 80;
+
+    access_log /var/log/nginx/access.log;
+    error_log /var/log/nginx/error.log;
+
+    location / {
+         rewrite /http://localhost:8110(.*) /$1  break;
```

Comment:
I',m not sure if you need the rewrite rule. It seems like you would like to pass all the parameters as they are, so proxy_pass should be enough. Please check that